src/StratisMasternodeDashboard/Controllers/HomeController.cs

```diff
@@ -116,49 +116,6 @@ public async Task<IActionResult> Index()
             return View("Dashboard", dashboardModel);
         }
 
-        /// <summary>
```

Comment:
Is where you call this in the JS already removed? If not remove any references to this method :)

Reply:
Yes, already removed Yesterday.